rootfs-builder/debian/Dockerfile-aarch64.in

```diff
@@ -0,0 +1,37 @@
+#
+# Copyright (c) 2020 ARM Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+
+# NOTE: OS_VERSION is set according to config.sh
+from docker.io/debian:@OS_VERSION@
+
+# RUN commands
+RUN apt-get update && apt-get install -y \
+    autoconf \
+    automake \
+    binutils \
+    build-essential \
+    chrony \
+    cmake \
+    coreutils \
+    curl \
+    debianutils \
+    debootstrap \
+    g++ \
+    gcc \
+    git \
+    libc-dev \
+    libstdc++-6-dev \
+    m4 \
+    make \
+    sed \
+    systemd \
+    tar \
+    vim \
+    wget
```

Comment:
Nit: I'm not sure we actually need `wget` (here and below) as we always download using `curl`...?

Reply:
oops,, sorry, I didn't give them a close look, just pasted them and delete the `musl` part. 
So I'll delete the `wget` here and test them again. ;)